docs/running.md

```diff
@@ -39,6 +39,28 @@ $ mpirun -np 16 \
     python train.py
 ```
 
+### Hangs due to SSH issues
+
+`mpirun` requires the host where it is executed to be able to SSH to all other hosts without any prompts.
```

Comment:
The host where `mpirun` is executed must be able to SSH to all other hosts without any prompts.